### PR TITLE
New check: me-> should be omitted

### PR DESCRIPTION
**Pull request by DennstedtB**
apparently I can't handle github correctly... I did delete the fork and closed the PR ( #786 ) before it was merged... 
So here again the new check # 94 ( it is the same like PR #786 )
 The check has a look if you access your static or member attributes with the classname or "me->" in front.

### CODE REVIEW

**Comment by DennstedtB** [comment]
Did rename everything, because in PR #783 there is already a check with number 94 - so I did rename everything to 95

File: src/checks/zcl_aoc_check_95.clas.abap [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+  ENDMETHOD.
+
+
+  METHOD check_instance_access.
+    DATA ls_token TYPE stokesx.
+    LOOP AT io_scan->tokens INTO ls_token FROM mv_impl_start_position.
+      TRY.
+          IF ls_token-str+0(4) = 'ME->'.
+            inform( p_test = me->myname                 " Name der Klassse
```
Comment (larshp) [inline src/checks/zcl_aoc_check_95.clas.abap]: "me->" should be omitted?

Comment (DennstedtB) [reply src/checks/zcl_aoc_check_95.clas.abap]: According to the [SAP Styleguide](https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#omit-the-self-reference-me-when-calling-an-instance-method) yes. But the user ( or whoever is the admin of the companys sci / atc ) has a setting if it should be checked or not. 
And for Static Attributes / Methods I referenced myself to your issue #468

Comment (DennstedtB) [reply src/checks/zcl_aoc_check_95.clas.abap]: And yes, "me->" should be omitted [Mentioned in the sap styleguide](https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#omit-the-self-reference-me-when-calling-an-instance-method)

Comment (larshp) [reply src/checks/zcl_aoc_check_95.clas.abap]: so "inform( p_test = me->myname" should be changed to "inform( p_test = myname"? in this code

Comment (DennstedtB) [reply src/checks/zcl_aoc_check_95.clas.abap]: .... oooops... :D see, if I had the check this wouldn't happen. I'll change in a second.

File: docs/_checks/95.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Object / Class member access
+cNumber: CHECK_95
+rfc: true
+---
+
+As mentioned in the SAP Styleguide, the "me->" self reference [should be omitted](https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#omit-the-self-reference-me-when-calling-an-instance-method)
+The reason, that "me->" is set implicit by the system does also valid for static member access or method calling.
```
[the remark this answers is not in the document]
Comment (DennstedtB) [reply docs/_checks/95.md]: done.

**Pull request merged**